MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -233,6 +233,29 @@
                 inlineUniformBlockProps->maxDescriptorSetUpdateAfterBindInlineUniformBlocks = _properties.limits.maxDescriptorSetUniformBuffers;
 				break;
 			}
+#if MVK_MACOS
+            case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SUBGROUP_PROPERTIES:
+                if (mvkOSVersionIsAtLeast(10.14)) {
+                    auto* subgroupProps = (VkPhysicalDeviceSubgroupProperties*)next;
+                    subgroupProps->subgroupSize = _metalFeatures.subgroupSize;
+                    subgroupProps->supportedStages =
+                        VK_SHADER_STAGE_COMPUTE_BIT |
+                        VK_SHADER_STAGE_TESSELLATION_CONTROL_BIT |
+                        VK_SHADER_STAGE_TESSELLATION_EVALUATION_BIT |
```

Comment:
This bit shouldn't be set, so just `COMPUTE | TESSELLATION_CONTROL | FRAGMENT`, then.